pkg/setup/setup.go

```diff
@@ -292,7 +292,11 @@ func (s *Setup) installDefaultPlugins(httpClient *httpclient.Client) error {
 	if errCore != nil {
 		return errCore
 	}
-	return errEnterprise
+
+	if errEnterprise != nil {
+		s.logger.Errorf("cluster setup was successful but dcos-enterprise-cli could not be installed: %s", errEnterprise)
```

Comment:
The log message should start with a capital letter. "Cluster setup was sucessful" is implicit with the exit code so could be removed.

Reply:
I added that to make it clear that while there was an error, the cluster was still set up properly. I know the error code will signal that but a normal user of the cluster probably won't check that especially if they're told something errored out.

That may not be the ideal place for something like that since it is a log message. An alternative could maybe be some final line printed to `ctx.Out()` which says essentially (setup was fine, enterprise failed) that and leaving the log line as just saying the enterprise cli install failed.